hardware/arduino/avr/cores/arduino/USBAPI.h

```diff
@@ -110,11 +110,15 @@ class Mouse_
 	Mouse_(void);
 	void begin(void);
 	void end(void);
+#ifdef MOUSE_ABS_ENABLED
+	void moveAbs(unsigned int x, unsigned int y, signed char wheel, unsigned char buttons); // x and y have the range of 0 to 4095
```

Comment:
For consistency with the Arduino API style guide (http://www.arduino.cc/en/Reference/APIStyleGuide), this should probably be "moveAbsolute()" not "moveAbs()".